Rubberduck.Core/UI/CodeExplorer/CodeExplorerControl.xaml

```diff
@@ -266,7 +266,12 @@
                 <Menu VerticalContentAlignment="Center" Background="Transparent" >
                     <MenuItem ItemsSource="{StaticResource AddModuleCommands}" VerticalContentAlignment="Center">
                         <MenuItem.Header>
-                            <Image Style="{StaticResource ToolbarIconStyle}" Source="{StaticResource AddModuleImage}" />
+                            <StackPanel Orientation="Horizontal" VerticalAlignment="Center">
+                                <Image Style="{StaticResource ToolbarIconStyle}" Source="{StaticResource AddModuleImage}" />
+                                <Border Margin="4,0,0,0">
```

Comment:
this margin is less than in the other cases...

Reply:
> the border margin when there's no label looks better at half the width vs when there's a label

Yeah... I'll just bite that bullet and template it with a left margin of `6`.